src/devices/HuskyLens/IBinaryConnection.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Iot.Device.HuskyLens
+{
+    /// <summary>
+    /// Represents the connection object used by the <see cref="HuskyLens"/> to communicate with the device
+    /// </summary>
+    public interface IBinaryConnection
```

Comment:
I would make it internal and hide this from the outside, see comment in the constructor of HuskyLens

Reply:
also you can use Stream instead